tests/cypress/integration/block.test.js

```diff
@@ -0,0 +1,39 @@
+describe("Test block functions", () => {
+	before(() => {
+		cy.login();
+	});
+
+	beforeEach(() => {
+		cy.mapsBlockSaveSettings();
+	});
```

Comment:
Are you expecting to add more tests here? 

The thought behind my question is this:
* there are only two tests
* in the second test this is re-run to simulate broken settings, this means it's only required for one test.

Reply:
I am not going to add more tests in this PR, just utilizing "initialize before" practice from Cypress recommendations [Clean up state before tests run](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks)